admin-manual/installation-setup/installation/installation.rst

```diff
@@ -0,0 +1,202 @@
+developed and is planned for a future release.
+
+
+Hardware
+--------
+
+Archivematica is capable of running on almost any hardware supported by Linux;
+however, processing large collections will require better hardware.
+
+Minimum hardware requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For small-scale functionality testing using small collections (transfers with 100
+files or less, total file size 1 GB or smaller), we recommend the following minimum
+hardware requirements:
+
+* Processor: 2 CPU cores
+* Memory: 2GB+
+* Disk space (processing): 7GB plus two to three times the disk space required for the
+  collection being processed (e.g., 3GB to process a 1GB transfer)
+
+Recommended minimum production requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For production processing, the hardware requirements depend almost entirely on
+the size and number of files being processed. These recommendations should be
+considered the minimum for a viable production system:
+
+* Processor: 2 CPU cores
+* Memory: 4GB
+* Disk space (processing): 200GB
+
+More commonly, we deploy the following:
+
+* Processor: 8 CPU cores
+* Memory: 16GB
+
+For processing disk space, we recommend allocating 20GB plus four times
+the disk space required for the largest transfer that you expect to process. If
+your largest transfer is 50GB, allocation at least 220GBs of disk space.
+
+The amount of transfer source disk space needed is subjective, and depends on
+individual workflows.
+
+The amount of storage disk space needed will depend on how much material you
+intend to store, as well as how it is stored (compressed or uncompressed).
+
+These requirements may not be suitable for certain types of material - for example,
+audio-visual material requires more processing power than images or documents.
+
+.. _instructions:
+
+Instructions for new installations
+==================================
+
+Archivematica can be installed using packages or Ansible scripts in either
+CentOS/Redhat or Ubuntu environments. It can also be installed using Docker.
+At this time, installation instructions are provided for officially tested and
+supported installation environments:
+
+* :ref:`Automated install from Github on Ubuntu (14.04 and 16.04) <ansible-git-ubuntu>`
+* :ref:`Manual install of OS packages on Ubuntu (14.04 and 16.04) <install-ubuntu>`
+* :ref:`Manual install of OS packages on CentOS/Redhat <install-pkg-centos>`
+
+Installing Archivematica using :ref:`Docker <docker>` is not officially
+supported for production deployments. However, it is the preferred development
+environment for those who work on Archivematica's code.
+
+For more information about installation environments, please see the
+`ansible-archivematica-src`_ repo, the `deploy-pub`_ repo, and ask on the
+`archivematica-tech`_ mailing list for more details.
+
+If you are upgrading from a previous version of Archviematica, please see the
```

Comment:
`s/Archviematica/Archivematica`

Reply:
Proposal: rename the whole project instead.

Reply:
lol